packages/react-native/ReactAndroid/src/test/java/com/facebook/react/uimanager/ReactPropForShadowNodeSpecTest.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.uimanager
+
+import android.view.View
+import com.facebook.react.uimanager.annotations.ReactProp
+import com.facebook.react.uimanager.annotations.ReactPropGroup
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+/**
+ * Test that verifies that spec of methods annotated with @ReactProp in {@link ReactShadowNode} is
+ * correct
+ */
+@RunWith(RobolectricTestRunner::class)
+@Ignore
+class ReactPropForShadowNodeSpecTest {
+
+  companion object {
```

Comment:
Could you please move the companion object to the end of the class?

Reply:
Moved